guide/components.html

```diff
@@ -18,59 +18,69 @@
 
     <!-- Load Customized St. Jude Bootstrap Theme -->
     <link href="css/styles.css" rel="stylesheet">
-
-    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
-    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
-    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
-    <script src="js/navbar.js"></script>
+    
   </head>
 
   <body id="home">
 
     <header>
-      <!-- Global Omnibar -->
+            <!-- Global Omnibar -->
       <div class="omnibar">
-        <nav class="navbar navbar-expand-lg">
+        <nav class="navbar">
           <a class="navbar-brand" href="https://stjude.cloud">
             <span class="logo">
-              <img src="images/stjude-logo-child.png" alt="St. Jude Cloud" title="St. Jude Cloud" />
+              <img src="../dist/images/stjude-logo-child.png" alt="St. Jude Cloud" title="St. Jude Cloud" />
             </span>
           </a>
 
           <span class="sjc-title">
             <a href="https://stjude.cloud">St. Jude Cloud</a>
           </span>
           <span class="site-title mr-auto">
-            <span>Designer and Developer Guide</span>
+            <span>App Title</span>

```

Comment:
This can stay the same

```suggestion
            <span>Designer and Developer Guide</span>
```